scalikejdbc-core/src/main/scala/scalikejdbc/Tx.scala

```diff
@@ -1,19 +1,38 @@
 package scalikejdbc
 
-import java.sql.{ SQLException, Connection }
+import java.sql.Connection._
+import java.sql.{ Connection, SQLException }
+
 import scala.util.control.Exception._
 import scala.util.control.NonFatal
 
 /**
  * DB Transaction abstraction.
  * @param conn connection
  */
-class Tx(val conn: Connection) {
+class Tx(val conn: Connection, isolationLevel: IsolationLevel = IsolationLevel.Default) {
```

Comment:
I'm not super happy about having a default here since it means that it's possible to forget "forwarding" the IsolationLevel parameter, but I guess the compatibility requirements kind of means that we have to do it this way.